apps/bridge-dapp/src/containers/UploadSpendNoteModal/PasteModalContent.tsx

```diff
@@ -66,10 +66,21 @@ export const PasteModalContent: FC<PasteModalContentProps> = ({
             key={id}
             value={note}
             onChange={(value) => {
-              setRawNotes((prev) => ({ ...prev, [id]: value }));
+              if (!isValidNote(value)) {
```

Comment:
Can we only check when a user clicks the upload button, if you check here, it can pollute the notification 🥲 

https://user-images.githubusercontent.com/60747384/219096561-3434aafa-9437-437a-9529-ed51f451c0c4.mp4

Reply:
@devpavan04 please address this concern and we should be good to merge.